04LessonFour/in-class-project/index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>The Daily Planet</title>
+  <title>Intel webpage</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <header>
+    <h1>Intel</h1>
+    <nav>
+        <a href-"#">i7 Processor</a>
+        <a href-"#">Solid State Drives</a>
+    </nav>
+  </header>
+  <main>

```

Comment:
You need more legibility here. Try adding vertical space, commenting, and then beautifying this!